### PR TITLE
Mosquitto 1.6.6 and 1.5.9

### PR DESCRIPTION
**Pull request by ralight**

### CODE REVIEW

File: library/eclipse-mosquitto [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Maintainers: Roger Light <roger@atchoo.org> (@ralight)
 GitRepo: https://github.com/eclipse/mosquitto.git
-GitCommit: 095bbb5e441e45538e924d90483dacb51d62b927
+GitCommit: 2e94e5999bc794afe618051176926ed67812106a
```
Comment (yosifkit) [inline library/eclipse-mosquitto]: This commit isn't on `master`, so you'll need a `GitFetch: refs/heads/fixes`.

Comment (ralight) [reply library/eclipse-mosquitto]: Sorry about that. I'll fix it tomorrow morning.

**Comment by ralight** [comment]
The commit is now on master, so this should be ok again.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index e5aacb5..faff573 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,5 +1,5 @@
 eclipse-mosquitto:1.5
-eclipse-mosquitto:1.5.8
+eclipse-mosquitto:1.5.9
 eclipse-mosquitto:1.6
-eclipse-mosquitto:1.6.5
+eclipse-mosquitto:1.6.6
 eclipse-mosquitto:latest
diff --git a/eclipse-mosquitto_1.5/Dockerfile b/eclipse-mosquitto_1.5/Dockerfile
index 28beaca..6e22463 100644
--- a/eclipse-mosquitto_1.5/Dockerfile
+++ b/eclipse-mosquitto_1.5/Dockerfile
@@ -3,8 +3,8 @@ FROM alpine:3.8
 LABEL maintainer="Roger Light <roger@atchoo.org>" \
     description="Eclipse Mosquitto MQTT Broker"
 
-ENV VERSION=1.5.8 \
-    DOWNLOAD_SHA256=78d7e70c3794dc3a1d484b4f2f8d3addebe9c2da3f5a1cebe557f7d13beb0da4 \
+ENV VERSION=1.5.9 \
+    DOWNLOAD_SHA256=d7b62aa0ca680b0d869d6883373903362f98326a6465fc6cd01a0b9e0e8f0333 \
     GPG_KEYS=A0D6EEA1DCAE49A635A3B2F0779B22DFB3E717B7 \
     LWS_VERSION=2.4.2
 
diff --git a/eclipse-mosquitto_latest/Dockerfile b/eclipse-mosquitto_latest/Dockerfile
index c8102ed..1982934 100644
--- a/eclipse-mosquitto_latest/Dockerfile
+++ b/eclipse-mosquitto_latest/Dockerfile
@@ -3,8 +3,8 @@ FROM alpine:3.8
 LABEL maintainer="Roger Light <roger@atchoo.org>" \
     description="Eclipse Mosquitto MQTT Broker"
 
-ENV VERSION=1.6.5 \
-    DOWNLOAD_SHA256=bc71b38b5a26fc7cc772853e5607c657868db9f9a6d2b15e2b677649a0f85d20 \
+ENV VERSION=1.6.6 \
+    DOWNLOAD_SHA256=82676bf4201ff102be1511b56b041a9450fbbfeda40b21aa28be0fee56e8de17 \
     GPG_KEYS=A0D6EEA1DCAE49A635A3B2F0779B22DFB3E717B7 \
     LWS_VERSION=2.4.2
 
```

</details>

**Comment by yosifkit** [comment]
Build test of #6648; 30f2c85a; `amd64` (`eclipse-mosquitto`):

```console

$ bashbrew build eclipse-mosquitto:1.6.6
Building bashbrew/cache:b2b34d4089e0a6507839f8711792685dc13acba88fe7cfb8f87c52fde7eba01b (eclipse-mosquitto:1.6.6)
Tagging eclipse-mosquitto:1.6.6
Tagging eclipse-mosquitto:1.6
Tagging eclipse-mosquitto:latest

$ test/run.sh eclipse-mosquitto:1.6.6
testing eclipse-mosquitto:1.6.6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build eclipse-mosquitto:1.5.9
Building bashbrew/cache:146b8868d563055cf39239f7375fc4563aab5cb4b409af1c64fea5e0d6bdcc05 (eclipse-mosquitto:1.5.9)
Tagging eclipse-mosquitto:1.5.9
Tagging eclipse-mosquitto:1.5

$ test/run.sh eclipse-mosquitto:1.5.9
testing eclipse-mosquitto:1.5.9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**